packages/angular/src/migrations/update-14-8-0/rename-webpack-server.ts

```diff
@@ -0,0 +1,26 @@
+import type { Tree } from '@nrwl/devkit';
+import { getProjects, updateProjectConfiguration } from '@nrwl/devkit';
+
+export default function renameWebpackServer(tree: Tree) {
+  const projects = getProjects(tree);
+
+  const oldExecutorName = '@nrwl/angular:webpack-server';
+  const newExecutorName = '@nrwl/angular:webpack-dev-server';
+
+  for (const [projectName, project] of projects.entries()) {
+    let stringifiedProject = JSON.stringify(project);
+    if (!stringifiedProject.includes(oldExecutorName)) {
+      continue;
+    }
+    for (const [targetName, target] of Object.entries(project.targets)) {
+      if (target.executor === oldExecutorName) {
+        project.targets[targetName] = {
+          ...project.targets[targetName],
+          executor: newExecutorName,
+        };
```

Comment:
Don't need to allocate a new object, `target` will hold a reference to the target in the `project`. We can set the executor directly:

```suggestion
        target.executor = newExecutorName;
```